Content.Server/Station/Systems/StationDampeningSystem.cs

```diff
@@ -17,12 +18,16 @@ private void OnInitStation(ref StationPostInitEvent ev)
             // If the station grid doesn't have defined dampening, give it a small dampening by default
             // This will ensure cargo tech pros won't fling the station 1000 megaparsec away from the galaxy
             if (!TryComp<PassiveDampeningComponent>(grid, out var dampening))
-            {
                 dampening = AddComp<PassiveDampeningComponent>(grid);
-                dampening.Enabled = true;
-                dampening.LinearDampening = 0.01f;
-                dampening.AngularDampening = 0.01f;
-            }
+
+            EntityManager.TryGetComponent(grid, out ShuttleComponent? shuttleComponent);
+
+            // PassiveDampeningComponent conflicts with shuttles cruise control a frontier QOL and is resetting dampeners causing issues.
+            // so if a station which shuttles have the station component too, then don't reset the physics to a near off state when it gets bumped
+            dampening.Enabled = true;
+            dampening.LinearDampening = shuttleComponent?.LinearDamping ?? 0.01f;
+            dampening.AngularDampening = shuttleComponent?.AngularDamping ?? 0.01f;
```

Comment:
So EE decided to randomly ignore the station-defined passive dapening?
```suggestion
            // Floofstation - changed this to NOT ignore station-defined dampening.
            // PassiveDampeningComponent conflicts with shuttles cruise control a frontier QOL and is resetting dampeners causing issues.
            // so if a station which shuttles have the station component too, then don't reset the physics to a near off state when it gets bumped
            if (TryComp(grid, out ShuttleComponent? shuttleComponent)) {
                dampening.Enabled = true;
                dampening.LinearDampening = shuttleComponent?.LinearDamping ?? 0.01f;
                dampening.AngularDampening = shuttleComponent?.AngularDamping ?? 0.01f;
            }
```

Reply:
I this this ignoring passive dampening is a brain child of solidus. The idea was, nothing stops in space, but that doesn't really work as we had lots of problems with that in the past. So here we go enable it again.